react/components/EditorContainer/Sidebar/index.tsx

```diff
@@ -50,9 +52,13 @@ const Sidebar: React.FunctionComponent<Props> = ({
         id="admin-sidebar"
         className="transition animated fadeIn b--light-silver bw1 z-2 h-100 pt8 pt0-ns calc--height-ns overflow-x-hidden w-100 font-display bg-white shadow-solid-x w-18em-ns admin-sidebar"
       >
-        <div className="h-100 flex flex-column dark-gray">
+        <div
+          className={`h-100 flex flex-column dark-gray ${
+            isLoading ? 'relative' : ''
```

Comment:
add a TODO to see in the future whether this conditional `relative` is necessary